ci_scripts/github_release.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+var=0
+final=""
+while IFS= read -r line; do
```

Comment:
rather than error prone reading line by line, you can also use git tag and sed example:

```
sed '/^## 0.6.0$/,/^## .*/!d;//d;/^$/d' ./CHANGELOG.md
```

Reply:
replace 0.6.0 with `"$1"` or `git describe --exact-match --tags $(git log -n1 --pretty='%h')`
and save that to a variable. That won't need separate script also, instead can be done on the makefile.

Reply:
Okay , yeah we can do it this way . Will do the changes asap @alexadhy @jdknives